test/com/minseungkwon/datastructure/Graph/BinaryTreeTest.java

```diff
@@ -2,12 +2,15 @@
 
 import org.junit.Test;
 
+import java.util.ArrayList;
+import java.util.LinkedList;
+
 import static org.junit.Assert.*;
 
 
 public class BinaryTreeTest {
-    public Integer[] makeArrayForTree() {
-        Integer[] arr = new Integer[10];
+    public Integer[] makeArrayForTree(int size) {
```

Comment:
You can overload this method by keep the previous method. Let the previous method call this new method with size of 10. This way you don't need to change all over the places. :)

Reply:
Okie